scripts/_run-aws-eks-commands.js

```diff
@@ -4,6 +4,101 @@ const yargs = require('yargs')
 const { execSync } = require('child_process')
 const { defaultProvider } = require('@aws-sdk/credential-provider-node')
 
+function commandExistsSync(cmd) {
+  try {
+    execSync(`command -v ${cmd}`)
```

Comment:
is this supported in windows?

check if os.type() is WINDOWS_NT then use where … 
which on other platforms

Reply:
We don't support windows :thinking:

Reply:
I made a survey on slack. As my feeling is that we need and should support Windows.